presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -49,35 +52,37 @@ public void runCommand()
             case RPM:
                 File tempFile;
                 try {
+                    LOGGER.debug("Downloading package from url : {}", urlToFetchPackage);
                     tempFile = createTempFile("presto", ".rpm");
                     copyURLToFile(urlToFetchPackage, tempFile, CONNECTION_TIMEOUT, READ_TIMEOUT);
                 }
                 catch (IOException e) {
+                    LOGGER.error("Failed to download package from url {}", urlToFetchPackage, e);
                     throw new PrestoManagerException(format("Failed to download file from url %s", urlToFetchPackage), e);
                 }
                 installUsingRpm(tempFile.toString(), disableDependencyChecking);
                 tempFile.delete();
                 break;
-            case TARBALL:
+            case GZ:
                 // TODO: Add tarball installation
                 throw new UnsupportedOperationException("Tarball installation is not supported");
             default:
-                // TODO: Add to logger
                 throw new IllegalArgumentException(format("Unsupported package type %s", packageType));
         }
     }
 
     private static void installUsingRpm(String pathToRpm, boolean checkDependencies)
             throws PrestoManagerException
     {
-        String checkRpm = format("rpm -K --nosignature %s", pathToRpm);
+        String checkRpm = format("rpm -Kv --nosignature %s", pathToRpm);
```

Comment:
oh you mean when you capture standard out.  I see.  I was confused because the logging wasn't happening in this method.